es-space-mgmt/main/default/lwc/customerList/customerList.js

```diff
@@ -11,14 +19,45 @@ export default class CustomerList extends LightningElement {
     msgForUser;
     wiredRecords;
 
-    @wire(CurrentPageReference) pageRef;
+    subscription = null;
+
+    @wire(MessageContext)
+    messageContext;
+
+    subscribeToMessageChannel() {
+        if (!this.subscription) {
+            this.subscription = subscribe(
+                this.messageContext,
+                FLOW_STATUS_CHANGE_MC,
+                (message) => this.handleMessage(message),
+                { scope: APPLICATION_SCOPE }
```

Comment:
I'm not super familiar with the app but do you need that scope or could you go with the default?
In other words, are you sending the message to other tabs?

Reply:
This comment also applies to the other two components that use LMS

Reply:
Application Scope is needed as the event is coming from an LWC inside a Flow.. Tried with default scope.. didnt work.

Reply:
ah good to know that because it is inside flow there is no way Active works. It does make sense now!